website/www/site/assets/js/quotes-slider.js

```diff
@@ -10,33 +10,32 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-var slider = new KeenSlider("#my-keen-slider", {
-  loop: true,
-  created: function (instance) {
-    var dots_wrapper = document.getElementById("dots");
-    var slides = document.querySelectorAll(".keen-slider__slide");
-    slides.forEach(function (t, idx) {
-      var dot = document.createElement("button");
-      dot.classList.add("dot");
-      dots_wrapper.appendChild(dot);
-      dot.addEventListener("click", function () {
-        instance.moveToSlide(idx);
-      });
-    });
-    updateClasses(instance);
-  },
-  slideChanged(instance) {
-    updateClasses(instance);
+$(document).ready(function(){
+  const SelectorNames = {
+    slider: 'slick-JS',
+    dotsWrapper: 'dots-wrapper',
+    dots: 'dots',
   }
-});
 
-function updateClasses(instance) {
-  var slide = instance.details().relativeSlide;
+  $(`.${SelectorNames.slider}`).slick({
```

Comment:
consider grouping bool values together and ditching the SelectorNames map 

`$('.slick-JS').slick({
    mobileFirst: true,
    infinite: true,
    arrows: false,
    dots: true,
    slidesToShow: 1,
    slidesToScroll: 1,
    appendDots: '.dots-wrapper',
    dotsClass: 'dots',
    responsive: [
      {
        breakpoint: 1024,
        settings: {
          infinite: false,
          slidesToShow: 3,
          slidesToScroll: 3,
        }
      },
    ]
  });`